Makefile

```diff
@@ -82,121 +15,21 @@ ci-shell: clean .dapper                  ## Launch a shell in the CI environment
-
-remote-debug-exit: bin/dlv               ## Kill dlv started with make remote-debug
-	echo exit | ./bin/dlv connect :2345 --init /dev/stdin
-
-.dev-shell-build: package-airgap
-	docker build -t ${PROG}-dev --target shell .
-
-clean-cache:                             ## Clean up docker base caches used for development
-	docker rm -fv ${PROG}-dev-shell
-	docker volume rm ${PROG}-cache ${PROG}-pkg
-
-clean:                                   ## Clean up workspace
-	rm -rf bin dist build
-
-dev-shell: .dev-shell-build              ## Launch a development shell to run test builds
-	docker run --rm --name ${PROG}-dev-shell --hostname ${PROG}-server -ti -e WORKSPACE=$$(pwd) -p 127.0.0.1:2345:2345 -v $${HOME}:$${HOME} -v ${PROG} -w $$(pwd) --privileged -v ${PROG}-pkg:/go/pkg -v ${PROG}-cache:/root/.cache/go-build ${PROG}-dev bash
-
-dev-shell-enter:                         ## Enter the development shell on another terminal
-	docker exec -it ${PROG}-dev-shell bash
-
-PEER ?= 1
-dev-peer: .dev-shell-build              ## Launch a server peer to run test builds
-	docker run --rm --link ${PROG}-dev-shell:${PROG}-server --name ${PROG}-peer${PEER} --hostname ${PROG}-peer${PEER} -p 127.0.0.1:234${PEER}:2345 -ti -e WORKSPACE=$$(pwd) -v $${HOME}:$${HOME} -v ${PROG} -w $$(pwd) --privileged -v ${PROG}-pkg:/go/pkg -v ${PROG}-cache:/root/.cache/go-build ${PROG}-dev bash
-
-dev-peer-enter:                         ## Enter the peer shell on another terminal
-	docker exec -it ${PROG}-peer${PEER} bash
-
-artifacts: build download-charts package-airgap
-	cp bin/${PROG} dist/artifacts/${RELEASE}
-
-.ci: validate-ci artifacts
+.ci: validate build package
 
 in-docker-%: .dapper                     ## Advanced: wraps any target in Docker environment, for example: in-docker-build-debug
 	mkdir -p ./bin/ ./dist/
 	./.dapper -f Dockerfile --target dapper make $*
 
-CHARTS_DIR = build/static/charts
-MANIFEST_DIR = manifests
-CHARTS = canal:v3.13.3 coredns:1.10.101 kube-proxy:v1.18.4 metrics-server:2.11.100 nginx-ingress:1.36.300
-download-charts:
-	mkdir -p $(CHARTS_DIR)
-	for chart in $(CHARTS); do \
-    	  chart_name=`echo "$${chart}" | cut -d ":" -f 1`;\
-    	  chart_version=`echo "$${chart}" | cut -d ":" -f 2`;\
-    	  curl -sfL https://dev-charts.rancher.io/$$chart_name/$$chart_name-$$chart_version.tgz -o ${CHARTS_DIR}/$$chart_name-$$chart_version.tgz;\
-    	  chart_content=`base64 -w 0 ${CHARTS_DIR}/$$chart_name-$$chart_version.tgz`;\
-    	  sed -e "s|%{CHART_CONTENT}%|$$chart_content|g" $(MANIFEST_DIR)/$$chart_name.yml > $(CHARTS_DIR)/$$chart_name-chart.yml;\
-    	  rm ${CHARTS_DIR}/$$chart_name-$$chart_version.tgz;\
-    done
-
 ./.dapper:
 	@echo Downloading dapper
 	@curl -sL https://releases.rancher.com/dapper/v0.5.0/dapper-$$(uname -s)-$$(uname -m) > .dapper.tmp
 	@@chmod +x .dapper.tmp
 	@./.dapper.tmp -v
 	@mv .dapper.tmp .dapper
 
-k8s-image: k8s-image-build k8s-image-scan
```

Comment:
Might be good to leave the dependencies in place and have separate build & scan scripts (for example).